extensions/GitExtension/FileExplorerGitIntegration/Strings/en-us/Resources.resw

```diff
@@ -161,4 +161,16 @@
     <value>Renamed, Modified</value>
     <comment>File status to display when its Git status is renamed with unstaged changes</comment>
   </data>
-</root>
+  <data name="SubmoduleStatusStaged" xml:space="preserve">
+    <value>Submodule staged</value>
+    <comment>Submodule status to display when a submodule is added and staged</comment>
+  </data>
+  <data name="SubmoduleStatusDirty" xml:space="preserve">
+    <value>Submodule dirty</value>
+    <comment>Submodule status to display when the Git submodule has local modifications</comment>
+  </data>
+  <data name="SubmoduleStatusChanged" xml:space="preserve">
+    <value>Submodule changed</value>
+    <comment>Submodule status to display when the Git submodule refers to a different commit.</comment>

```

Comment:
minor nit: remove period for consistency with other comments added to file

Reply:
Fixed.